src/fs/bare/file.ts

```diff
@@ -6,12 +6,12 @@ import BaseFile from '../base/file'
 export class BareFile extends BaseFile {
 
   static create(content: FileContent): BareFile {
-    return new BareFile(content)
+    return new BareFile(content) // why not `super`?
```

Comment:
how do you mean?

Reply:
> Outdated

Sorry, I had copy/pasted, and then changed it to

> return new BareFile(content) // why not the constructor?